designsafe/apps/auth/tasks.py

```diff
@@ -47,21 +51,26 @@ def check_or_configure_system_and_user_directory(username, system_id, path):
                     f"Checked and there is a need to configure system:{system_id} path:{path} for {username}: {e}")
 
     try:
-        tg458981_client = get_tg458981_client()
-        try:
-            # User tg account to check if path exists
-            tg458981_client.files.listFiles(systemId=system_id, path=path)
-            logger.info(f"Home directory for user={username} on system={system_id}/{path} exists and works. ")
-        except NotFoundError:
-            logger.info("Creating the home directory for user=%s then going to run setfacl on system=%s", username,
-                        system_id)
-
-            tg458981_client.files.mkdir(systemId=system_id, path=path)
-            tg458981_client.files.setFacl(systemId=system_id,
-                                          path=path,
-                                          operation="ADD",
-                                          recursionMethod="PHYSICAL",
-                                          aclString=f"d:u:{username}:rwX,u:{username}:rwX")
+        if create_path:
+            tg458981_client = get_tg458981_client()
+            try:
+                # User tg account to check if path exists
+                tg458981_client.files.listFiles(systemId=system_id, path=path)
```

Comment:
We should also `try / catch` for the `cloud.data` system. I think here we can also be using the user's client to do the listing for all systems

Reply:
The listing of using the user's client occurs on line 39.  If that fails, then we start the process of checking if the directory/path needs to be created/configured (optional) and setting up credentials.

Here (line 58), we're just seeing if the path exists and if doesn't, we create and set acls.

Reply:
Oops you're right, all good - resolving